app/assets/javascripts/compute_resource.js

```diff
@@ -6,8 +6,9 @@ $(function() {
       if (status == "error") {
         $(this).closest(".tab-content").find("#spinner").html(Jed.sprintf(__("There was an error listing VMs: %(status)s %(statusText)s"), {status: xhr.status, statusText: xhr.statusText}));
       }
-      $('.dropdown-toggle').dropdown();
-      $(document.body).trigger('ContentLoad');
+      else {
+        activateDatatables();
```

Comment:
no need to actually run the entire `onContentLoad` function, we just want to activate the datatables here.